lib/import_js/emacs_editor.rb

```diff
@@ -0,0 +1,145 @@
+# encoding: utf-8
+module ImportJS
+end
```

Comment:
It depends on how you load this class, but there's some code in `lib/import_js.rb` that should make sure the module is never empty.